boards/posix/native_posix/doc/index.rst

```diff
@@ -516,6 +516,10 @@ The following peripherals are currently provided with this board:
   must be powered down and support Bluetooth Low Energy (i.e. support the
   Bluetooth specification version 4.0 or greater).
 
+**USB controller**:
+  It's possible to use Virtual USB controller working over USB/IP protocol.
```

Comment:
Would an article would be missing here : "..to use +the+ Virtual USB.."

Reply:
Done